src/item_group.cpp

```diff
@@ -174,7 +178,7 @@ item Single_item_creator::create_single( const time_point &birthday, RecursionLi
 {
     item tmp = create_single_without_container( birthday, rec );
     if( container_item ) {
-        tmp = tmp.in_container( *container_item, tmp.count(), sealed );
+        tmp = tmp.in_container( *container_item, tmp.count(), sealed, *container_item_variant );
```

Comment:
Phone review, so may have missed something, but I don't see any reason to know `container_item_variant.has_value()`
```suggestion
        tmp = tmp.in_container( *container_item, tmp.count(), sealed, container_item_variant.value_or( "" ) );
```